### PR TITLE
native: small-fix omnibus

### PR DESCRIPTION
**Pull request by jamesacklin**
- Adds additional icon and label states to InviteFriendsToTlonButton (fixes TLON-2881).
- Adds an "Invite people" button to ChannelOptions in the correct administrative/privacy contexts (fixes TLON-2880).
- Moves the "Next" button in SetTelemetryScreen from the header to a full-width button in the screen content (fixes TLON-2883).
- Always show InviteFriendsToTlonButton in EmptyChannelNotice for admins on subsequent visits to an empty channel (fixes TLON-2879).
- Adds padding to the top of form field validation messages (fixes TLON-2815).
- Fixes an issue where screen elements were too wide in group creation (fixes TLON-2877).

### CODE REVIEW

File: packages/ui/src/components/ChatOptionsSheet.tsx [hunk 1]
```diff
@@ -635,6 +639,43 @@ export function ChannelOptions({
             } as ActionGroup,
           ]
         : []),
+      ...(group &&
+      !['groupDm', 'dm'].includes(channel.type) &&
```
Comment (patosullivan) [inline packages/ui/src/components/ChatOptionsSheet.tsx:643]: This logic looks right, we might wanna define this in an easier to read way elsewhere, though.

File: packages/ui/src/components/ChatOptionsSheet.tsx [hunk 2]
```diff
@@ -635,6 +639,43 @@ export function ChannelOptions({
             } as ActionGroup,
           ]
         : []),
+      ...(group &&
```
Comment (patosullivan) [inline packages/ui/src/components/ChatOptionsSheet.tsx:642]:
```suggestion
     // TODO: redefine in a more readable way.
      ...(group &&
```

**Review by patosullivan — APPROVED** [review]
Looks good! One nit about the ternary expressions in ChatOptionsSheet, but not blocking.

File: packages/ui/src/components/ChatOptionsSheet.tsx (hunk 1 above)
Comment (jamesacklin) [reply packages/ui/src/components/ChatOptionsSheet.tsx:644]: "canInvite," I guess. I'll make a note and follow up later.

**Pull request merged**